geode-core/src/main/java/org/apache/geode/internal/InternalDataSerializer.java

```diff
@@ -1506,7 +1506,9 @@ public static void writeDSFID(DataSerializableFixedID o, DataOutput out) throws
       // error condition, so you also need to check to see if the JVM
       // is still usable:
       SystemFailure.checkFailure();
+      t.printStackTrace();
```

Comment:
Remove this printStackTrace call. Since "t" is given to ToDataException it seems like that should be enough. If you do need something here use the logger but I would advise against that and leave it up to the handler of the ToDataException to display the cause.

Reply:
done - thanks!